### PR TITLE
Minor Bug Fixes

### PR DESCRIPTION
**Pull request by kankurodev**
Trimmed server boost time to match joined and created time formats
Fixed masked urls within message edit notifications
Fixed join/leave user mention
Fixed excluded channels

### CODE REVIEW

File: controllers/MessageController.js [hunk 1]
```diff
@@ -71,14 +71,16 @@ module.exports = {
                 const channelExcluded = excluded_trigger_channels.some(name => message.channel.name.includes(name));
                 
                 // If within an excluded channel then ignore
-                if(channelExcluded) {
+                if(channelExcluded === true) {
```
Comment (safaci2000) [inline controllers/MessageController.js:74]: why is this needed?

Comment (safaci2000) [reply controllers/MessageController.js:74]: isn't if(channelExcluded) enough?

Comment (kankurodev) [reply controllers/MessageController.js:74]: Yea, usually, but unsure why it wasn't working on the actual server, so I figured I'd throw is in as a failsafe, feel free to remove it.
